pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -280,6 +281,23 @@ def test_load_compartment():
         check_dtype=False,
     )
 
+    # test using non-default float_datatype
+    loaded_compartment_df = AP.load_compartment(
+        compartment="cells", float_datatype=np.float32
+    )
+    pd.testing.assert_frame_equal(
+        loaded_compartment_df,
+        CELLS_DF.astype(
```

Comment:
This chaining is a bit hard to follow. Can you create a new df (`CELLS_float32_DF` or something), and make sure it is a deep copy? (see https://github.com/cytomining/pycytominer/pull/241#discussion_r1028571680)

Reply:
I have the same comment for the second test

Reply:
Understood and will do, thank you for the comment to address this. I'll add an update to this test to make certain it's addressed.